core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala

```diff
@@ -53,7 +55,8 @@ object RuntimeConfigRequest {
                                   workerDiskSize: Option[DiskSize] = None, //min 10
                                   numberOfWorkerLocalSSDs: Option[Int] = None, //min 0 max 8
                                   numberOfPreemptibleWorkers: Option[Int] = None,
-                                  properties: Map[String, String])
+                                  properties: Map[String, String],
+                                  zone: Option[ZoneName] = None)
```

Comment:
So Dataproc accepts a region _or_ a zone (unlike GCE which only accepts a zone). See: https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/auto-zone

I'm wondering if we should have both in our API to cover both cases, e.g.:
```
region: Option[RegionName] = None,
zone: Option[ZoneName] = None
```

(And actually I don't see in this PR where the Dataproc configs are used in `DataprocInterpreter.scala`.)

Reply:
Or maybe we _only_ support region for Dataproc